apps/Gruntfile.js

```diff
@@ -573,6 +684,14 @@ module.exports = function (grunt) {
     'karma:integration'
   ]);
 
+  grunt.registerTask('codeStudioTest', [
+    'newer:messages',
+    'exec:convertScssVars',
+    'newer:copy:static',
+    'concat',
+    'karma:codeStudio'
+  ]);
```

Comment:
Any reason we're not just lumping the (fairly small) code studio test suite into unitTests?